docs/running/wdl.rst

```diff
@@ -0,0 +1,139 @@
+WDL Support in Toil
+********
+
+Support is still in the alpha phase and should be able to handle basic wdl files.  See the specification below for more
+details.
+
+How to Run a WDL file in Toil
+-----------
+Recommended best practice when running wdl files is to first use the Broad's wdltool for syntax validation and generating the
+needed json input file.  Full documentation can be found on the repository_, and a precompiled jar binary can be downloaded here: wdltool_.
```

Comment:
Clearly Java is a requirement. This  should be more explicit with the version needed to run this successfully.

Reply:
Fixed.